lib/nodes/Func.js

```diff
@@ -101,7 +103,10 @@ class Func extends Container {
     parser.back(rightTokens);
 
     const { lastNode } = parser;
-    lastNode.isColor = colorFunctions.includes(lastNode.name.toLowerCase());
+    const lowerName = lastNode.name.toLowerCase();
+    const { nodes } = node;
+    lastNode.isColor = colorFunctions.includes(lowerName);
+    lastNode.isVar = lowerName === 'var' && nodes.length && reVar.test(nodes[0].value);
```

Comment:
In theory no need for custom regex to test for var.
But variable detection could be configured to not include CSS custom properties by `{ variables: { prefixed: ['\\$'] }}`.